ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -90,6 +90,21 @@ import RevenueCat
         Purchases.logLevel = level
     }
 
+    /**
+     * Sets a log handler and forwards all logs to completion function.
+     *
+     * - Parameter completion: Gets a map with two keys, a `logLevel` which  is one of the ``LogLevel``  name lowercased, and a `message`, with the log message.
+     */
+    @objc public static func setLogHander(completion: @escaping ([String: String]) -> Void) {
+        Purchases.logHandler = { logLevel, message in
+            let logDetails = [
+                "logLevel": logLevel.description.uppercased(),
+                "message": message
+            ]
+            completion(logDetails)
```

Comment:
Hmm I think we should rename the lambda since it's not really a completion but a `callback` or an `onLogReceived` or something like that.

Reply:
👍🏻